terraform/main.ts

```diff
@@ -263,6 +265,74 @@ class VargasJRInfrastructureStack extends TerraformStack {
     return "https://www.vargasjr.dev/api/ses/webhook";
   }
 
+  private createVargasJRAgentRole(tags: Record<string, string>) {
+    const vargasJRAgentRole = new IamRole(this, "VargasJRAgentRole", {
+      name: "vargasjr-agent",
+      assumeRolePolicy: JSON.stringify({
+        Version: "2012-10-17",
+        Statement: [
+          {
+            Action: "sts:AssumeRole",
+            Effect: "Allow",
+            Principal: {
+              AWS: "*", // This will be restricted by external ID or other conditions in practice
+            },
+            Condition: {
+              StringEquals: {
+                "sts:ExternalId": "vargasjr-agent-external-id"
```

Comment:
I realized I made a mistake and it makes no sense to do sts bc it won't be an AWS resource assuming this role. Create a `vargasjr-vellum` user that will have this role attached